CHAGELOG-template.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+_These releases can contain markup changes and potentially breaking changes.
+A :warning: icon indicates a change that might be a breaking change for your project._
+
+## [Version #]
+<!-- 
+Every release should have an entry.
+Releases should be listed in reverse chronological order, with the newest release listed first.
+-->
+
+[Release date]
+<!-- 
+Include the date the version was released.
+The date should be in the following format: January 1, 2000. -->
+
+[Itemized list of all user-affected changes]
+<!--
+Example format:
+```
+- [Brief statement that summarizes the change] (#[Related PR])
+  - [Optional 1-2 sentence description included in a nested list item.]
+  - [:warning: Summary of breaking or markup updates 
+    with explanation of what action the user must take
+    These items must be flagged with a :warning: icon at the start of the statement.]
+```
+Whenever possible, change items should:
```

Comment:
Spacing or additional formatting to separate this or make it stand out.

Reply:
Added a blank line in https://github.com/uswds/uswds-team/pull/207/commits/9bd41cdf4efbfb2d3fe3c3606b5712e87fccf49e